Project_JS/index.html

```
@@ -0,0 +1,119 @@
+                <textarea id="arrayManipulationTextArea" type="text" disabled>Output in console</textarea>
+            </div>
+            <div class="concept">
+                <p>Object Manipulation</p>
+                <button onclick="handleObjectManipulation()">Run</button>
+                 <textarea name="" id="objectManipulationTextArea" disabled></textarea>
+            </div>
+        </div>
+
+    </div>
+
+    </div>
```

HTMLHint (1.5.0)

[error] Tag must be paired, no start tag: [ </div> ]

(tag-pair)